pandas/core/frame.py

```diff
@@ -2554,6 +2554,11 @@ def to_feather(self, path: FilePathOrBuffer[AnyStr], **kwargs) -> None:
         """
         Write a DataFrame to the binary Feather format.
 
+        This function writes the dataframe as a `feather file
```

Comment:
put *below* the Parameters section
its called
```
Notes
-------
<text>
```

Reply:
Oh, didn't see that at all. Thanks.